src/dosbox.cpp

```diff
@@ -666,15 +666,28 @@ void DOSBOX_Init()
 	pstring->Set_help("CPU core used in emulation ('auto' by default). 'auto' will switch to dynamic\n"
 	                  "if available and appropriate.");
 
-	const char* cputype_values[] = { "auto", "386", "386_slow", "486_slow", "pentium_slow",
+	const char* cputype_values[] = { "386dx", "486dx", "pentium",
 #if C_MMX
-	                                  "pentium_mmx",
+	                                 "pentium_mmx",
 #endif
-	                                  "386_prefetch", nullptr};
-	pstring = secprop->Add_string("cputype", always, "auto");
+	                                 "386dx_prefetch", "auto", nullptr};
+#if C_MMX
+	pstring = secprop->Add_string("cputype", always, "pentium_mmx");
+	pstring->Set_values(cputype_values);
+	pstring->Set_help(
+	        "CPU type used in emulation ('pentium_mmx' by default).\n"
+#else
+	pstring = secprop->Add_string("cputype", always, "pentium");
 	pstring->Set_values(cputype_values);
 	pstring->Set_help(
-	        "CPU type used in emulation ('auto' by default). 'auto' is the fastest choice.");
+	        "CPU type used in emulation ('pentium' by default).\n"
+#endif
+	        "Prefetch variant emulates the prefetch queue and requires the 'normal' core.");
+	pstring->SetDeprecatedWithAlternateValue("386", "386dx");
```

Comment:
Should be `386dx_prefetch`.

Reply:
According to your suggestion above, I got rid of DX postfix.